src/libraries/Common/src/System/Obsoletions.cs

```diff
@@ -42,5 +42,8 @@ internal static class Obsoletions
 
         internal const string CodeBaseMessage = "Assembly.CodeBase and Assembly.EscapedCodeBase are only included for .NET Framework compatibility. Use Assembly.Location instead.";
         internal const string CodeBaseDiagId = "SYSLIB0012";
+
+        internal const string WebRequestMessage = "Use HttpClient instead.";
```

Comment:
We [just updated](https://github.com/dotnet/runtime/commit/cb2023c5fc31d447227cf21eb360e356b79284e8) the `CodeBaseMessage` to provide more context on the message.  Please expand this message in a similar fashion.  This will of course need to be updated throughout the ref assemblies as well.

The need for this becomes more obvious when adding the the obsoletion to the [list of obsoletions](https://github.com/dotnet/runtime/blob/master/docs/project/list-of-obsoletions.md), which also needs to be done.